README.md

```diff
@@ -66,6 +66,7 @@ The compatibility matrix with dependencies is as follows:
 
 | Libhdbpp-Cassandra Version | Cassandra Version | Datastax C++ Driver | Libuv |
 |---------|-------------------|------------|-------|
+| 0.11.0 | 2.2.9 | 2.2.1 | 1.4.2 |
```

Comment:
It would be good to be able to relate the version with the library file name. For instance with this version this file name is libhdb++cassandra.so.7.1.0.